src/components/Pensjonsavtaler/OffentligTjenestePensjon/OffentligTjenestepensjon.tsx

```diff
@@ -176,16 +175,16 @@ export const OffentligTjenestepensjon = (props: {
                 size="xsmall"
               >
                 {intl.formatMessage({
-                  id: 'pensjonsavtaler.offentligtp.subtitle.spk',
+                  id: leverandørMessageKeyMap[tpLeverandør],
```

Comment:
Vi bruker ikke æ ø å i kodebasen. Foreslår derfor"leverandoerMessageKeyMap" slik at det blir konsekvent. Det kan også være lurt å ha dette adskilt i utils fil for å få komponentfilen mindre.